ross/utils.py

```diff
@@ -4,6 +4,23 @@
 from bokeh.models import LogColorMapper
 from bokeh.palettes import Viridis256
 from bokeh.plotting import figure
+import itertools
+from collections import Iterable
+
+from ross.materials import Material
+
+def stochastic_ross(f,*args):
```

Comment:
Please add the function documentation. Also check if we could improve the name of this function to give a better idea of its purpose to the user.